Specs/DateTime/Korean/HolidayParser.json

```diff
@@ -4,7 +4,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "NotSupported": "dotnet",
+    "IgnoreResolution": "true",
```

Comment:
Please later add comments on why the resolution is not passing.

Reply:
As per agreement via email at the beginning of the work, for this round of implementation, we are not looking into the resolution for Korean and Arabic, but only for Japanese.